### PR TITLE
Document the automated release process

### PR DESCRIPTION
**Pull request by anweshadas**
Adding the automated release process via Github action.

### CODE REVIEW

File: docs/community-ee/community-ee-release-process.md [hunk 1]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
 
-- Access to  https://github.com/ansible-community/images repo
-- Access to ghcr.io
+    - Access to  https://github.com/ansible-community/images repo
+    - Access to ghcr.io
 
+## Pre Requisite:
 
-## Build steps for Community-ee-base
+      - Join [Release Management working Matrix room](https://app.element.io/?updated=1.11.38#/room/#release-management:ansible.com) and [Execution Envs group in Forum Group](https://forum.ansible.com/g/ExecutionEnvs).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:26]:
```suggestion
      - Join the [Release Management working group](https://forum.ansible.com/g/release-managers) and [Execution Environment group](https://forum.ansible.com/g/ExecutionEnvs).
```

File: docs/community-ee/community-ee-release-process.md [hunk 2]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Read about the Ansible Execution Environment in [here](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341) and in [this](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341) Forum Post.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:27]:
```suggestion
      - Understand [Ansible execution environments](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341).
```

File: docs/community-ee/community-ee-release-process.md [hunk 3]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Read about  Github workflow `.github/workflows/eerelease.yml` .
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:28]:
```suggestion
      - Read about the [eerelease.yml](/.github/workflows/eerelease.yml)  GitHub workflow.
```

File: docs/community-ee/community-ee-release-process.md [hunk 4]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Show intention and book the date for the releasing of the EE.
 
-1. Go to the working directory.
+## Announcement before the release
 
-`cd images/execution-environments/community-ee-base`
+  Post this message in the channel before working on the release.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:33]:
```suggestion
  Post this message in the channel before working on the release, using the correct release number.
```

File: docs/community-ee/community-ee-release-process.md [hunk 5]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
 
-2. Create a new release branch
+  ```
+  📯📯📯 I am working on the `Ansible Execution Environment Base and Minimal 2.17.1-1` Release. I will keep the room updated about the progress.📦️
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:36]:
```suggestion
  📯📯📯 I am working on the `Ansible Execution Environment Base and Minimal 2.x.y-1` Release. I will keep the room updated about the progress.📦️
```

File: docs/community-ee/community-ee-release-process.md [hunk 6]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+  ```
 
-`git checkout -b base-<ansible-core-version-base-ee-version>`
+## Actual Build Steps :
 
-3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and [ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps) and edit accordingly.
+### On the terminal
 
-`vim execution-environments/community-ee-base/execution-environment.yml`
+      - Check the ansible-core version [here](https://pypi.org/project/ansible-core/)
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:43]:
```suggestion
      - Check the most recent [ansible-core version](https://pypi.org/project/ansible-core/)
```

File: docs/community-ee/community-ee-release-process.md [hunk 7]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Verify ansible collection versions for ansible.posix, ansible.utils and  ansible.windows for `deps` file of the ansible version for the related release  [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:44]:
```suggestion
      - Verify ansible collection versions for ``ansible.posix``, ``ansible.utils`` and  ``ansible.windows`` for `deps` file of the Ansible community package version for the related release  in the [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
```

File: docs/community-ee/community-ee-release-process.md [hunk 8]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
-git commit
-git push origin <ansible-core-version-minimal-ee-version>
-```
-Compare and create the pull request.
-
-11.   Check the community-ee-minimal images [here](https://github.com/orgs/ansible-community/packages/container/package/community-ee-minimal)  and get the sha256 sum.
+      - Go to the [ansible-community/images](https://github.com/ansible-community/images) repo
+      - Click on the ![`Actions`](action-workflow-block.png) to open the `Workflows`
+      - Go to the `Release Ansible Excution Environment` and click on the ![`Run Workflow` Dropdown Button](docs/community-ee/dropdown-workflow.png)
+      - Click on the Drop down Button and choose the type of the Execution Environment, (whether it is `community-ee-base` and `community-ee-minimal`)
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:74]:
```suggestion
      - Click on the Drop down Button and choose the type of the Execution Environment, (whether it is `community-ee-base` or `community-ee-minimal`)
```

File: docs/community-ee/community-ee-release-process.md [hunk 9]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Write down the version of execution environment (to be appended to the ansible-core version) .
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:75]:
```suggestion
      - Write down the patch number of execution environment (to be appended to the ansible-core version) .
      This is typically 1.
```

File: docs/community-ee/community-ee-release-process.md [hunk 10]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      -  Check on the button if this is `latest` release of  the particular execution environment
+      -  Click on the ![`Run workflow` button](docs/community-ee/workflow_block.png) to run the workflow
+      - After the successful run of the workflow, go on and check if the image is published in  [here](https://github.com/orgs/ansible-community/packages/container) and get the sha256 sum of the published image (to be used in the announcement).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:78]:
```suggestion
      - After the successful run of the workflow, go on and check if the [image is published](https://github.com/orgs/ansible-community/packages/container) and get the sha256 sum of the published image (to be used in the announcement).
```

**Review by samccann — COMMENTED** [review]
urls shouldn't ever be [here] or [this], etc. Always best to use descriptive text. See https://developers.google.com/search/docs/crawling-indexing/links-crawlable#write-good-anchor-text if you want to understand why.

File: docs/community-ee/community-ee-release-process.md [hunk 11]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
 
 ##  Credentials
 
-- Access to  https://github.com/ansible-community/images repo
-- Access to ghcr.io
+    - Access to  https://github.com/ansible-community/images repo
+    - Access to ghcr.io
 
+## Pre Requisite:
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:24]:
```suggestion
## Prerequisites
```

File: docs/community-ee/community-ee-release-process.md [hunk 12]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
-4. List images in local storage
+        ```
 
-`podman images`
+      - Update the ansible-core and collection versions in the  `/images/execution-environments/community-ee-base/execution-environment.yml` file.
+      - Update the ansible-core version in the  `/images/execution-environments/community-ee-minimal/execution-environment.yml` file, commit the changes and create a PR.
 
-5. Delete the existing community-ee base and fedora images
+        ```
+            vim community-ee-base/execution-environment.yml`
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:60]:
```suggestion
            vim community-ee-base/execution-environment.yml
```

Comment (oraNod) [reply docs/community-ee/community-ee-release-process.md:60]: Maybe it's unnecessary to suggest vim here. You risk opening up the vim vs emacs can of worms. People get attached to their editors. You could just say "Open `community-ee-base/execution-environment.yml` for editing and then do:

File: docs/community-ee/community-ee-release-process.md [hunk 13]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+            git add -u
+            git status
+            git commit -m "Updates ansible-core & collection versions for Base and Minimal"
+            git push origin branch_name
+        ```
+      - Once the PR is merged, it is the time to start with the github action workflow.
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:66]:
```suggestion
      - Once the PR is merged, it is the time to start with the github workflow.
```

File: docs/community-ee/community-ee-release-process.md [hunk 14]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
-podman rmi <podman_community-ee-fedora_IMAGE_ID>
-```
-6. Create a virtual environment
 
-```
-python3 -m venv .venv
-source .venv/bin/activate
-```
-7. Install `ansible-builder` and `setuptools`
+### On your browser :
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:69]:
```suggestion
### Triggering the release workflow
```

File: docs/community-ee/community-ee-release-process.md [hunk 15]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
-
-```
-git add execution-environments/community-ee-minimal/execution-environment.yml
-git commit
-git push origin <ansible-core-version-minimal-ee-version>
-```
-Compare and create the pull request.
-
-11.   Check the community-ee-minimal images [here](https://github.com/orgs/ansible-community/packages/container/package/community-ee-minimal)  and get the sha256 sum.
+      - Go to the [ansible-community/images](https://github.com/ansible-community/images) repo
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:71]:
```suggestion
      - Go to the [ansible-community/images](https://github.com/ansible-community/images) repo.
```

File: docs/community-ee/community-ee-release-process.md [hunk 16]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Click on the ![`Actions`](action-workflow-block.png) to open the `Workflows`
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:72]:
```suggestion
      - Click on the ![`Actions`](action-workflow-block.png) tab to view the available workflows.
```

Comment (oraNod) [reply docs/community-ee/community-ee-release-process.md:72]: I'd recommend removing the screengrabs here. We don't really need to document a third-party UI.

File: docs/community-ee/community-ee-release-process.md [hunk 17]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Go to the `Release Ansible Excution Environment` and click on the ![`Run Workflow` Dropdown Button](docs/community-ee/dropdown-workflow.png)
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:73]:
```suggestion
      - Go to the `Release Ansible Execution Environment` workflow and then select ![`Run Workflow`](docs/community-ee/dropdown-workflow.png).
```

File: docs/community-ee/community-ee-release-process.md [hunk 18]
```diff
@@ -18,145 +18,63 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Click on the Drop down Button and choose the type of the Execution Environment, (whether it is `community-ee-base` and `community-ee-minimal`)
+      - Write down the version of execution environment (to be appended to the ansible-core version) .
+      -  Check on the button if this is `latest` release of  the particular execution environment
+      -  Click on the ![`Run workflow` button](docs/community-ee/workflow_block.png) to run the workflow
+      - After the successful run of the workflow, go on and check if the image is published in  [here](https://github.com/orgs/ansible-community/packages/container) and get the sha256 sum of the published image (to be used in the announcement).
+      - Make the announcement in the #release-management and #community-working-group Matrix room, Forum and Bullhorn following the `docs/community-ee/community-ee-announcement.md` file.
```
Comment (oraNod) [inline docs/community-ee/community-ee-release-process.md:79]:
```suggestion
      - Make the announcement in the #release-management and #community-working-group Matrix room, Forum, and Bullhorn according to the instructions in `docs/community-ee/community-ee-announcement.md`.
```

**Comment by anweshadas** [comment]
> urls shouldn't ever be [here] or [this], etc. Always best to use descriptive text. See https://developers.google.com/search/docs/crawling-indexing/links-crawlable#write-good-anchor-text if you want to understand why.

Thank you for pointing it out, I will follow it for the future while documenting something.

File: docs/community-ee/community-ee-release-process.md [hunk 19]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
 
-- Access to  https://github.com/ansible-community/images repo
-- Access to ghcr.io
+    - Access to  https://github.com/ansible-community/images repo
+    - Access to ghcr.io
 
+## Prerequisites
 
-## Build steps for Community-ee-base
+      - Join the [Release Management working group](https://forum.ansible.com/g/release-managers) and [Execution Environment group](https://forum.ansible.com/g/ExecutionEnvs).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:26]:
```suggestion
- Join the [Release Management working group](https://forum.ansible.com/g/release-managers) and [Execution Environment group](https://forum.ansible.com/g/ExecutionEnvs).
```

File: docs/community-ee/community-ee-release-process.md [hunk 20]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Understand [Ansible execution environments](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:27]:
```suggestion
- Understand [Ansible execution environments](https://forum.ansible.com/t/execution-environments-getting-started-guide-community-ee-images-availability/1341).
```

File: docs/community-ee/community-ee-release-process.md [hunk 21]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Read about the [eerelease.yml](/.github/workflows/eerelease.yml)  GitHub workflow.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:28]:
```suggestion
- Read about the [eerelease.yml](/.github/workflows/eerelease.yml)  GitHub workflow.
```

File: docs/community-ee/community-ee-release-process.md [hunk 22]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Show intention and book the date for the releasing of the EE.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:29]:
```suggestion
- Show intention and book the date for the releasing of the EE.
```

File: docs/community-ee/community-ee-release-process.md [hunk 23]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+  ```
 
-`git checkout -b base-<ansible-core-version-base-ee-version>`
+## Actual Build Steps :
 
-3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and [ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps) and edit accordingly.
+### On the terminal
 
-`vim execution-environments/community-ee-base/execution-environment.yml`
+      - Check the most recent [ansible-core version](https://pypi.org/project/ansible-core/)
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:43]:
```suggestion
1. Check the most recent [ansible-core version](https://pypi.org/project/ansible-core/)
```

File: docs/community-ee/community-ee-release-process.md [hunk 24]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Verify ansible collection versions for ``ansible.posix``, ``ansible.utils`` and  ``ansible.windows`` for `deps` file of the Ansible community package version for the related release  in the [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:44]:
```suggestion
2. Verify ansible collection versions for ``ansible.posix``, ``ansible.utils`` and  ``ansible.windows`` for `deps` file of the Ansible community package version for the related release  in the [ansible-build-data repo](https://github.com/ansible-community/ansible-build-data).
```

File: docs/community-ee/community-ee-release-process.md [hunk 25]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Go to the images/execution-environments directory.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:45]:
```suggestion
3. Go to the images/execution-environments directory.
```

File: docs/community-ee/community-ee-release-process.md [hunk 26]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Create the correct `git branch` (name it in the related version eg: 2.17.1-1)
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:46]:
```suggestion
4. Create the correct `git branch` (name it in the related version eg: 2.17.1-1)
```

File: docs/community-ee/community-ee-release-process.md [hunk 27]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+            cd images/execution-environments
+            git checkout main
+            git pull upstream main
+            git checkout -b coreversion-eeversion
 
-4. List images in local storage
+        ```
 
-`podman images`
+      - Update the ansible-core and collection versions in the  `/images/execution-environments/community-ee-base/execution-environment.yml` file.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:56]:
```suggestion
5. Update the ansible-core and collection versions in the  `/images/execution-environments/community-ee-base/execution-environment.yml` file.
```

File: docs/community-ee/community-ee-release-process.md [hunk 28]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
 
-5. Delete the existing community-ee base and fedora images
+        ```
+            vim community-ee-base/execution-environment.yml
+            git add -u
+            git status
+            git commit -m "Updates ansible-core & collection versions for Base and Minimal"
+            git push origin branch_name
+        ```
+      - Once the PR is merged, it is the time to start with the github workflow.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:66]:
```suggestion
Once the PR is merged, it is the time to start with the github workflow.
```

File: docs/community-ee/community-ee-release-process.md [hunk 29]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
-
-```
-git add execution-environments/community-ee-minimal/execution-environment.yml
-git commit
-git push origin <ansible-core-version-minimal-ee-version>
-```
-Compare and create the pull request.
-
-11.   Check the community-ee-minimal images [here](https://github.com/orgs/ansible-community/packages/container/package/community-ee-minimal)  and get the sha256 sum.
+      - Go to the [ansible-community/images](https://github.com/ansible-community/images) repo.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:71]:
```suggestion
1. Go to the [ansible-community/images](https://github.com/ansible-community/images) repo.
```

File: docs/community-ee/community-ee-release-process.md [hunk 30]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Click on the ![`Actions`](action-workflow-block.png) tab to view the available workflows.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:72]:
```suggestion
2. Click on the ![`Actions`](action-workflow-block.png) tab to view the available workflows.
```

File: docs/community-ee/community-ee-release-process.md [hunk 31]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Go to the `Release Ansible Execution Environment` workflow and then select ![`Run Workflow`](docs/community-ee/dropdown-workflow.png).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:73]:
```suggestion
3. Go to the `Release Ansible Execution Environment` workflow and then select ![`Run Workflow`](docs/community-ee/dropdown-workflow.png).
```

File: docs/community-ee/community-ee-release-process.md [hunk 32]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Click on the Drop down Button and choose the type of the Execution Environment, (whether it is `community-ee-base` or `community-ee-minimal`)
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:74]:
```suggestion
4. Click on the Drop down Button and choose the type of the Execution Environment, (whether it is `community-ee-base` or `community-ee-minimal`)
```

File: docs/community-ee/community-ee-release-process.md [hunk 33]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Write down the patch number of execution environment (to be appended to the ansible-core version) .
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:75]:
```suggestion
5. Write down the patch number of execution environment (to be appended to the ansible-core version) .
```

File: docs/community-ee/community-ee-release-process.md [hunk 34]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      This is typically 1.
+      -  Check on the button if this is `latest` release of  the particular execution environment
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:77]:
```suggestion
6. Check on the button if this is `latest` release of  the particular execution environment
```

File: docs/community-ee/community-ee-release-process.md [hunk 35]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      -  Click on the ![`Run workflow` button](docs/community-ee/workflow_block.png) to run the workflow
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:78]:
```suggestion
7. Click on the ![`Run workflow` button](docs/community-ee/workflow_block.png) to run the workflow
```

File: docs/community-ee/community-ee-release-process.md [hunk 36]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - After the successful run of the workflow, go on and check if the [image is published](https://github.com/orgs/ansible-community/packages/container) and get the sha256 sum of the published image (to be used in the announcement).
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:79]:
```suggestion
8. After the successful run of the workflow, go on and check if the [image is published](https://github.com/orgs/ansible-community/packages/container) and get the sha256 sum of the published image (to be used in the announcement).
```

File: docs/community-ee/community-ee-release-process.md [hunk 37]
```diff
@@ -18,145 +18,64 @@ Versioning would be the core tag + a patch number, e.g
...
+      - Make the announcement in the #release-management and #community-working-group Matrix room, Forum, and Bullhorn according to the instructions in `docs/community-ee/community-ee-announcement.md`.
```
Comment (samccann) [inline docs/community-ee/community-ee-release-process.md:80]:
```suggestion
9. Make the announcement in the #release-management and #community-working-group Matrix room, Forum, and Bullhorn according to the instructions in `docs/community-ee/community-ee-announcement.md`.
```

**Review by samccann — APPROVED** [review]

**Comment by oraNod** [comment]
@anweshadas and @samccann I also pushed some changes directly.

**Comment by samccann** [comment]
Thanks everyone! Merged as i think it's basically there. We can open PRs as needed to update from here.

**Pull request merged**